docker-compose.yml

```diff
@@ -7,14 +7,14 @@ services:
       context: .
       dockerfile: database/Dockerfile.centos
     ports:
-      - 5432:5432
+      - 15432:5432
```

Comment:
Why the port is changing?

Reply:
Because when we're running with VMaaS, it uses the 5432 port, this remaps the port only on host.